math/pythagoras.cpp

```diff
@@ -0,0 +1,28 @@
+#include <math.h>
+
+#include <iostream>
+
+// Our class for point coordinates.
+struct  Point {
+    int x;
+    int y;
+    int z;
+};
+
+// This method just calculate pythagorian distance from a point to b.
+int distance(Point a, Point b) {
+    int x_2 = pow(abs(b.x - a.x), 2);
```

Comment:
replace `pow`, `sqrt`, `abs` with their `std::` equivalents
`std::abs` is defined in `<algorithm>` header